webextensions/api/tabs.json

```
@@ -3890,7 +3883,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/tabs/setZoomSettings",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "38"
```

:x: Same as `getZoom`.

```suggestion
                "version_added": "42"
```